tests/test_views.py

```diff
@@ -1624,6 +1628,7 @@ def setUp(self):
         self.project = ProjectFactory(creator=self.creator)
         # Authenticate all requests
         self.app.authenticate(*self.creator.auth)
+        contributor_added.connect(notify_added_contributor)
```

Comment:
If `DISCONNECTED_SIGNALS` is made to be a class constant, this won't be necessary.